### ISSUE TITLE
Incompatible shapes at the mel_outputs tensor

### ISSUE THREAD

**Issue by chapter544** [keithito/tacotron#55]
Hi,
Thank you for such a clean implementation of Tacotron. I tried to train the network using my own speech data, but I got this strange error. From the code, those mismatch numbers are T_out. Can you give me some pointers? Thanks.

Exiting due to exception: Incompatible shapes: [32,1005,80] vs. [32,1000,80]
         [[Node: model/loss/sub = Sub[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/gpu:0"](datafeeder/input_queue_Dequeue/_555, model/inference/Reshape)]]
         [[Node: model/inference/post_cbhg/dense/Tensordot/Shape/_1109 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/cpu:0", send_device="/job:localhost/replica:0/task:0/gpu:0", send_device_incarnation=1, tensor_name="edge_11312_model/inference/post_cbhg/dense/Tensordot/Shape", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/cpu:0"]()]]

Caused by op 'model/loss/sub', defined at:
  File "train.py", line 154, in <module>
    main()
  File "train.py", line 150, in main
    train(log_dir, args)
  File "train.py", line 66, in train
    model.add_loss()
  File "/home/TTS/tacotron/models/tacotron.py", line 125, in add_loss
    self.mel_loss = tf.reduce_mean(tf.abs(self.mel_targets - self.mel_outputs))

**Comment by keithito** [comment]
You can fix this by setting the max_iters hyperparameter to a larger value, for example by adding the flag --hparams="max_iters=400".

**Comment by keithito** [comment]
See #16 for details.

**Comment by chapter544** [comment]
Thank you, I was able to increase the max_iters to fix the issue.
